src/sbt-test/sbt-us/simple/build.sbt

```diff
@@ -0,0 +1,7 @@
+ThisBuild / ubiquitousScaladoc / usSourceHtmlDir := file("api")
+ThisBuild / ubiquitousScaladoc / usTargetMarkdownDir := file("customTarget")
+
+lazy val root = (project in file("."))
+  .settings(
+    Compile / doc / target := baseDirectory.value / "api/generated"

```

Comment:
Use the DSL for directory separation `Compile / doc / target := baseDirectory.value / "api" / "generated"`